### ISSUE TITLE
favicon icon for the website

### ISSUE THREAD

**Issue by vattytrivedi** [UdacityFrontEndScholarship/udabuddy#118]

**Comment by Karthik781** [comment]
What icon to be used?

**Comment by vattytrivedi** [comment]
@Karthik781 Have you finalize the icon?

**Comment by vattytrivedi** [comment]
Here is the ico file.
https://drive.google.com/file/d/1GYbg532AH9CdLrZdQywvSp7DHQJ2Ucdc/view?usp=sharing
Sorry for the delay @Karthik781

**Comment by souryavaranasi** [comment]
@littlestar642  exactly! That would look amazing, the one with white bg.

**Comment by Karthik781** [comment]
@littlestar642 @souryavaranasi they look great but i suggest adding "udabuddy" tag under the logo. Anyway, it will do as plain as well.

**Comment by littlestar642** [comment]
Do someone want to take up this task of adding this icon in the navbar of the pages. @Karthik781 @vattytrivedi @souryavaranasi

**Comment by vattytrivedi** [comment]
Which type of logo you wish? @littlestar642

**Comment by littlestar642** [comment]
@vattytrivedi i guess the favicon can also serve as a good logo for the app presently

**Comment by littlestar642** [comment]
@souryavaranasi sure, please go ahead!

**Comment by vattytrivedi** [comment]
Is the logo finalized?
Or you have or need a new logo/concept?

**Comment by littlestar642** [comment]
@vattytrivedi Yep, We will be using this logo for the time. Also i am waiting for @Karthik781 to make a PR for the favicon

**Comment by littlestar642** [comment]
@Karthik781 Any ETA on this?

**Comment by seethaR03** [comment]
Is final icon decided or still open for suggestions?
